src/test/java/com/openlattice/rehearsal/edm/EdmTest.kt

```diff
@@ -60,10 +60,10 @@ class EdmTest : MultipleAuthenticatedUsersBase() {
         val es = createEntitySet(EdmTestConstants.personEt)
 
         entitySetsApi.addEntitySetsToLinkingEntitySet(linkingEs.id, setOf<UUID>(es.id))
-        Assert.assertEquals(es.id, entitySetsApi.getEntitySet(linkingEs.id).linkedEntitySets.single())
+        Assert.assertEquals(es.id, edmApi.getEntitySet(linkingEs.id).linkedEntitySets.single())
```

Comment:
These should also be entitySetsApi I think

Reply:
Yeah, the wrong version got merged